meta-resin-common/recipes-connectivity/redsocks/redsocks_0.5.bb

```diff
@@ -0,0 +1,17 @@
+DESCRIPTION = "redsocks - transparent socks redirector"
+SECTION = "net/misc"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRCREV = "27b17889a43e32b0c1162514d00967e6967d41bb"
+
+SRC_URI = "git://github.com/darkk/redsocks.git"
```

Comment:
We could patch redsocks to always not detach form the console . Should as simple as overriding it here: https://github.com/darkk/redsocks/blob/78a73fca15f4676c42d5dcc209484c40de2c6e33/main.c#L123 . In this way we can remove the requirement of daemon=off in the configuration.